src/Console/Command/BuildCommand.php

```diff
@@ -47,6 +47,7 @@ protected function configure()
                 new InputOption('repository-url', null, InputOption::VALUE_OPTIONAL, 'Only update the repository at given url', null),
                 new InputOption('no-html-output', null, InputOption::VALUE_NONE, 'Turn off HTML view'),
                 new InputOption('skip-errors', null, InputOption::VALUE_NONE, 'Skip Download or Archive errors'),
+                new InputOption('best-candidate-strategy', null, InputOption::VALUE_NONE, 'Download only the best candidate version'),
```

Comment:
I agree, but I was looking for a simple way to do it. The config file scare me a little and I don't know if it's a good idea or not add more configuration to the config file.